SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -64,27 +83,31 @@ def eval(self, f):
     def evalDeriv(self, f):
         raise Exception('Use Sources to project fields deriv.')
 
-#################
+#########
 # Data
-#################
+#########
 
 
-class Data(SimPEGsurvey.Data):
-    '''
+class Data(SimPEGsurvey.Data, DataNSEMPlotMethods):
+    """
     Data class for NSEMdata. Stores the data vector indexed by the survey.
 
     :param SimPEG.EM.NSEM.SurveyNSEM survey: NSEM survey object
     :param numpy.ndarray v: Vector of the data in order matching of the survey
+    :param numpy.ndarray standard_deviation: Vector of the standard_deviation
+        of data in order matching of the survey
+    :param numpy.ndarray floor: Vector of the noise floor of the data in
+        order matching of the survey
 
-    '''
-    def __init__(self, survey, v=None):
+    """
+    def __init__(self, survey, v=None, standard_deviation=None, floor=None):
         # Pass the variables to the "parent" method
-        SimPEGsurvey.Data.__init__(self, survey, v)
+        SimPEGsurvey.Data.__init__(self, survey, v, standard_deviation, floor)
 
 
-    def toRecArray(self,returnType='RealImag'):
+    def toRecArray(self, returnType='RealImag'):
```

Comment:
@grosenkj: I might get you to walk me through RecArrays on Tuesday if you are willing